src/analysis/normed_space/lp_space.lean

```diff
@@ -675,6 +731,40 @@ instance infty_smul_comm_class {𝕜} [normed_field 𝕜] [Π i, normed_space 
   smul_comm_class 𝕜 (lp B ∞) (lp B ∞) :=
 ⟨λ r f g, lp.ext $ smul_comm r ⇑f ⇑g⟩
 
+section star_ring
+
+variables [Π i, star_ring (B i)] [Π i, normed_star_group (B i)]
+
+instance : star_ring (lp B ∞) :=
+{ star_mul := λ f g, by {ext, simp only [lp.star_apply, infty_coe_fn_mul, pi.mul_apply, star_mul]},
+  .. (show star_add_monoid (lp B ∞),
+      by { letI : Π i, star_add_monoid (B i) := λ i, infer_instance, apply_instance }) }
+
+instance [∀ i, cstar_ring (B i)] : cstar_ring (lp B ∞) :=
+{ norm_star_mul_self :=
+begin
+intro f,
+apply le_antisymm,
+{     rw ←sq,
+      apply lp.norm_le_of_forall_le (sq_nonneg ∥ f ∥),
+      dsimp [lp.star_apply],
+      intro i,
+      rw [cstar_ring.norm_star_mul_self, ←sq],
+      refine sq_le_sq' _ _,
+      { linarith [norm_nonneg (f i), norm_nonneg f] },
+      { refine lp.norm_apply_le_norm ennreal.top_ne_zero _ _,}, },
```

Comment:
but since you have only one exact you can remove it entirely:
```suggestion
      refine sq_le_sq' _ (lp.norm_apply_le_norm ennreal.top_ne_zero _ _),
      linarith [norm_nonneg (f i), norm_nonneg f],
```